src/bin/add/main.rs

```diff
@@ -97,6 +98,9 @@ fn print_msg(dep: &Dependency, section: &[String], optional: bool) -> Result<()>
     if optional {
         write!(output, " optional")?;
     }
+    if let Some(f) = features {
+        write!(output, " features {}", f)?
```

Comment:
Could you place this output after [line 109](https://github.com/killercup/cargo-edit/pull/194/files#diff-4568dca113da4e44470302888eb5f360R109) (and change `writeln` to `write`), so that the output would be
```plain
... to dependencies with features: jui
```
rather than
```plain
to features jui dependencies
```